layouts/shortcodes/generate-ref-docs.html

```diff
@@ -0,0 +1,22 @@
+ {{ with $.Params.url }}
+  {{ $json := getJSON $.Params.url }}
+      <p>{{ $json.description }}</p> 
+       {{ range $key, $value := $json.properties }}
+          {{ with $value }}
+          <ul>
+          {{ partial "field-desc-ref-docs" (dict "value" $value "key" $key "url" $.Params.url) }}
+          <ul>
+          {{ range $childk, $childv := $value.properties }}
+              {{ with $childv }}
+                  {{ partial "field-desc-ref-docs" (dict "value" $childv "key" $childk) }}
+              {{ end }}
+          {{ end }}
+          {{ with $value.allOf }}
+               {{ partial "conditional-field-desc-ref-docs" (dict "value" $value "key" $key "url" $.Params.url) }}
+          {{ end }}
+          </ul>      
+       </ul>
+       {{ end }}
+       {{ end }}
+       </ul>      
+{{ end }}
```

Comment:
Please indent 4 spaces where needed ... some have less ... and other lines have missing indent 
(For increased readability)

Reply:
```suggestion
{{ with $.Params.url }}
    {{ $json := getJSON $.Params.url }}
    <p>{{ $json.description }}</p> 
    {{ range $key, $value := $json.properties }}
        {{ with $value }}
            <ul>
                {{ partial "field-desc-ref-docs" (dict "value" $value "key" $key "url" $.Params.url) }}
                <ul>
                {{ range $childk, $childv := $value.properties }}
                    {{ with $childv }}
                        {{ partial "field-desc-ref-docs" (dict "value" $childv "key" $childk) }}
                    {{ end }}
                {{ end }}
                {{ with $value.allOf }}
                    {{ partial "conditional-field-desc-ref-docs" (dict "value" $value "key" $key "url" $.Params.url) }}
                {{ end }}
                </ul>      
           </ul>
       {{ end }}
   {{ end }}
   </ul>      
{{ end }}
```

Reply:
Take note that there's an additional `</ul>`

Reply:
Thanks. Think I was aiming for 3, which is habit from markdown.